kong/init.lua

```diff
@@ -1052,7 +1052,7 @@ do
     }
 
     local res = ngx.location.capture("/kong_buffered_http", options)
-    if res.truncated then
+    if res.truncated and options.method ~= ngx.HTTP_HEAD then
```

Comment:
Could we add a test that `HEAD` call works with `response`?

Reply:
done!